app/controllers/sms_controller.rb

```diff
@@ -40,7 +40,7 @@ def create
     end
 
     # Mission is guaranteed to be set by this point.
-    if params[:token] != current_mission.setting.incoming_sms_token
+    unless [current_mission.setting.incoming_sms_token, configatron.universal_sms_token].include? params[:token]
```

Comment:
What if setting.incoming_sms_token is nil and params[:token] is nil?. Perhaps add a compact?